airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -59,21 +86,21 @@ def __init__(self,
                  cmds,
                  arguments,
                  name,
+                 secrets=None,
                  in_cluster=False,
                  labels=None,
                  startup_timeout_seconds=120,
-                 kube_executor_config=None,
                  get_logs=True,
                  *args,
                  **kwargs):
         super(KubernetesPodOperator, self).__init__(*args, **kwargs)
-        self.kube_executor_config = kube_executor_config or {}
```

Comment:
Why is this removed?

Reply:
In the past it seems that was a parameter for `make_pod`. I have no clue more about this but now it is not being used in this class so it's better remove the code

Reply:
It was removed in this commit https://github.com/apache/incubator-airflow/commit/e30a1f451aa5ec5aca4c886067ba8946a3d33395